src/script.js

```diff
@@ -0,0 +1,215 @@
+/* global ymaps */
+
+import regeneratorRuntime from "regenerator-runtime"; // eslint-disable-line
+import css from "./style.css"; // eslint-disable-line
```

Comment:
https://stackoverflow.com/questions/44474484/how-to-import-css-files-into-webpack